src/Filters/molecules/FilterDropdown.tsx

```diff
@@ -89,6 +90,15 @@ const EmptyResultText = styled.div`
   font-size: 12px;
 `;
 
+const Gradient = styled.div<{hasSearchOption: boolean}>`
+  position: absolute;
+  right: 0px;
+  top: ${({ hasSearchOption }) => hasSearchOption ? '223px' : '149px'};
```

Comment:
Wouldn't this be simpler to do using `bottom` instead? That should negate the need for a distance variable and will mean that if we want to vary height in the future, the gradient will move with it.

Reply:
> Wouldn't this be simpler to do using `bottom` instead? That should negate the need for a distance variable and will mean that if we want to vary height in the future, the gradient will move with it.

Done. Thanks for the feedback.